db/seeds.rb

```diff
@@ -15,6 +15,9 @@
 
 include SeedHelper
 
+# Defer guideline start date to allow users to become members w/o completing that.
+ENV['SHF_MEMBERSHIP_GUIDELINES_CHECKLIST_REQUIRED_START_DATE'] = Time.zone.tomorrow.to_s
```

Comment:
If you need to change the value, change it in your `.env` files (`.env.development`, `.env` etc.).  That's why it's an ENV value.

Seeding is happening correctly as is.